website/source/docs/configuration/providers.html.md

```diff
@@ -117,3 +117,14 @@ KEY {
 	CONFIG
 }
 ```
+
+## Interpolation
+Providers support [interpolation syntax](/docs/configuration/interpolation.html) allowing dynamic configuration at run time.
+
+```
+provider "aws" {
+	region = "${var.aws_region}"
+}
+```
+
+Only [variables](/docs/configuration/variables) and [remote state](/docs/state/remote.html) are supported, it is not possible to use the output from a resource, module or data source in the interpolation syntax for a provider.
```

Comment:
> and [remote state](/docs/state/remote.html) are supported

Not sure what exactly do you mean by this. 🤔 

Also I'd try to be a bit more positive in the wording - i.e. `are supported` **`at this point`** (or _currently_). There's at least one open issue about this and it's something we eventually want to support:
https://github.com/hashicorp/terraform/issues/4149